Mux-Stats-Google-IMA-Kaltura.podspec

```diff
@@ -14,23 +14,33 @@ Pod::Spec.new do |s|
                        s.author           = { 'Mux' => 'ios-sdk@mux.com' }
                        s.source           = { :git => 'https://github.com/kaltura/playkit-ios-ima.git', :tag => "v#{s.version}" }
 
-  s.ios.deployment_target = '10.0'
-  s.tvos.deployment_target = '10.0'
+  s.ios.deployment_target = '11.0'
```

Comment:
what required this change?

Reply:
the 'PlayKit_IMA' dependency that I had forgotten to add before it seems like that sdk only supports > iOS 11.0